rfcs/react-components/convergence/provider-style-overrides.md

```diff
@@ -0,0 +1,132 @@
+# RFC: Provider style overrides
+
+@ling1726
+
+## Summary
+
+Proposes two methods of overriding the default styles on the `FluentProvider`
+
+## Problem statement
+
+The `FluentProvider` ships with [default styles](https://github.com/microsoft/fluentui/blob/656f002f3062c0a7962fb23eb165dec64ecc1509/packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderStyles.ts#L11-L18) which can be too opinionated for some customers, especially those who have
+strict requirements on global styles in their application.
+
+Currently it's not possible to use `makeStyles` to override these default styles. See the example below:
```

Comment:
It's not entirely true, I suggest to change it:

```suggestion
If `makeStyles` will be used to override these default styles it will result in unexpected issues until additional wrapping will be done. See the example below:
```